crates/bevy_input/src/mouse.rs

```diff
@@ -155,3 +156,70 @@ pub fn mouse_button_input_system(
         }
     }
 }
+
+/// Tracks how much the mouse has moved every frame
+///
+/// This event tracks the `MouseMotion` event for calculations.
+#[derive(Resource, Debug, Clone, Copy, PartialEq, Reflect, Default)]
+#[reflect(Debug, PartialEq)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+pub struct AccumulatedMouseMotion {
+    /// The change in mouse position.
+    pub delta: Vec2,
+}
+
+/// Tracks how much the mouse has scrolled every frame
+///
+/// This event tracks the `MouseWheel` event for calculations.
+#[derive(Resource, Debug, Clone, Copy, PartialEq, Reflect)]
+#[reflect(Debug, PartialEq)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+pub struct AccumulatedMouseScroll {
+    /// The mouse scroll unit.
```

Comment:
This needs a warning explaining what happens if the mouse scroll unit changes.

Reply:
I added a warning explaining what could happen if you change the scroll unit while scrolling, but not while scrolling has paused. Is my new comment fine?

Reply:
Yep looks solid.